o1vm/src/mips/witness.rs

```diff
@@ -44,7 +45,9 @@ pub const NUM_INSTRUCTION_LOOKUP_TERMS: usize = 5;
 pub const NUM_LOOKUP_TERMS: usize =
     NUM_GLOBAL_LOOKUP_TERMS + NUM_DECODING_LOOKUP_TERMS + NUM_INSTRUCTION_LOOKUP_TERMS;
 // TODO: Delete and use a vector instead
-pub const SCRATCH_SIZE: usize = 97; // MIPS + hash_counter + byte_counter + eof + num_bytes_read + chunk + bytes + length + has_n_bytes
+// MIPS + hash_counter + byte_counter + eof + num_bytes_read + chunk + bytes
+// + length + has_n_bytes + chunk_bytes + preimage
+pub const SCRATCH_SIZE: usize = 98;
```

Comment:
the comment says you're adding two things, chunk_bytes + preimage, and yet the size only grows by one

Reply:
It's just the preimage key that is being added wrt what's in master. The problem is that in the old comment the chunk_bytes were not written down, but they were in any case part of the state https://github.com/o1-labs/proof-systems/blob/7d51847c00ff790bc22af3c83ddedc1deb070e38/o1vm/src/mips/column.rs#L26C1-L27C1